qt/widgets/common/src/WorkspacePresenter/WorkspaceTreeWidget.cpp

```diff
@@ -110,18 +110,28 @@ void WorkspaceTreeWidget::setupWidgetLayout() {
 
   auto *buttonLayout = new FlowLayout();
   m_loadButton = new QPushButton("Load");
+  m_loadButton->setToolTip("Load a file or live data");
   m_saveButton = new QPushButton("Save");
+  m_saveButton->setToolTip("Save the selected workspaces");
   m_deleteButton = new QPushButton("Delete");
+  m_deleteButton->setToolTip("Deletes the selected workspaces");
+  m_clearButton = new QPushButton("Clear");
+  m_clearButton->setToolTip("Deletes all workspaces");
   m_groupButton = new QPushButton("Group");
+  m_groupButton->setToolTip("Group together two or more selected workspaces");
```

Comment:
This is a funnier button, if you click on a group, the label transforms into Ungroup.
So should do the tooltip.

Reply:
Ah thanks! I didn't realise this